extensions/pyproject.toml

```diff
@@ -14,7 +14,7 @@ sphinx-copybutton = "^0.5.2"
 enum-tools = { extras = ["sphinx"], version = "^0.12.0" }
 furo = "^2024.8.6"
 myst-nb = "^1.1.1"
-
+sphinxcontrib_googleanalytics = "^0.4.0"
```

Comment:
Instead of editting the `pyproject.toml` file directly can you please use `poetry add sphinxcontrib.googleanalytics` this will ensure the `poetry.lock` file is correctly updated too.

Reply:
https://github.com/CQCL/pytket-docs-theming/pull/10/commits/02cc2e51908deeff62243fdbe56633da6fb64a84